### ISSUE TITLE
Support vector tiles output through pg_tileserv

### ISSUE THREAD

**Issue by AbelVM** [CrunchyData/pg_featureserv#40]
It would be great to add an extra output format/endpoint: **vectortiles**, embedding [pg_tileserv](https://github.com/CrunchyData/pg_tileserv) functionality within pg_featureserv. 

This way pg_featureserv would be the all-in-one weapon of choice to expose geospatial data :smiley:

**Comment by dr-jts** [comment]
We're trying to adhere to the microservices paradigm, rather than building a large monolithic server.  Keep the service code small and focussed makes development cycles shorter, and allows more flexibility for deployment.

Is there any reason you can't just run both services?

**Comment by AbelVM** [comment]
None, indeed. I just feel it makes sense to add this functionality to pg_featurerv

As they already share most of the endpoints defnition, query params and db connection and management logic, the inclusion of this feature looks more like adding a new format to the output than adding a new service. 

Just my two cents.

**Comment by dr-jts** [comment]
Yes, there is a fair bit of commonality.  We have talked about the relative merits of merging them versus keeping them separate.  Another option we've discussed is building a framework for the common parts, which can then be used for both services.

You are right that using MVT allows rendering bigger datasets than GeoJSON.  The use of GeoJSON in the pg_featureserv UI is deliberate, however.  It allows exercising the service and understanding the performance characteristics of feature queries.  So we will keep that aspect of the design.
